server/planning/__init__.py

```diff
@@ -114,6 +118,49 @@ def init_app(app):
     app.client_config['street_map_url'] = get_street_map_url(app)
     app.client_config['max_multi_day_event_duration'] = get_event_max_multi_day_duration(app)
 
+    # Set up Celery task options
+    if not app.config.get('CELERY_TASK_ROUTES'):
+        app.config['CELERY_TASK_ROUTES'] = CTR
+
+    if not app.config.get('CELERY_TASK_ROUTES').get('planning.flag_expired'):
```

Comment:
if `app.config['PLANNING_EXPIRY_MINUTES'] == 0`, then the command `planning:flag_expired` does nothing.

It might be good here to not add it to the message queue.